python-sdk/dev/Dockerfile

```diff
@@ -6,7 +6,13 @@ RUN apt-get install -y --no-install-recommends \
         build-essential \
         libsasl2-2 \
         libsasl2-dev \
-        libsasl2-modules
+        libsasl2-modules \
+        freetds-dev \
+        libssl-dev
+
+RUN apt-get update \
+ && apt-get install libkrb5-dev -y
```

Comment:
Why is this on separate line and not part of L11 ?

```
RUN apt-get install -y --no-install-recommends \
        build-essential \
        libsasl2-2 \
        libsasl2-dev \
        libsasl2-modules \
        freetds-dev \
        libssl-dev \
        libkrb5-dev
```

Reply:
We need the update to run before installing libkrb5-dev ( after installing libssl-dev and freetds-dev)

Reply:
Tested with your suggestion again today and it worked. So I have addressed your suggestion @kaxil